data/vendor-customer-support.ini

```diff
@@ -0,0 +1,8 @@
+[Customer Support]
```

Comment:
On the final system, the file would contain translations regardless of where they live, unless I'm missing something? So other programs could still use them.
We manage translations globally in Transifex, so digging into different po files would not be a concern.

We can leave it like this for now, but I'm tagging @rshuler for his preference here, since he's the one usually managing translations.